src/util/iterative_restoration.jl

```diff
@@ -1,108 +1,166 @@
+## Process::
+# 1. take network and create XX repair periods and solve ROP.
+# 2. take each single network from 1 and break into XX repair periods and solve ROP.
+#       - this requires extensive rehashing of the data dict to set status values
+#       - if a device is repaired after this network, the status is 0 ->seems to be set by solution builder? need to double check buses.
+#       - if a device is repaired before this network, the status is 1, damaged is 0
+#       - if a device is repaired in this network, the status is 1, damaged is 1
+# 3. repeat 2 until the number of repairs is <= XX and there is only 1 item repaired per time period
+# 4. report a final restoration dictionary with the ordered repairs (this dictionary should only contain
+#     solution data from last period restoration)
+
+
 "solve restoration using iterative period length"
-function run_iterative_restoration(network_data, model_constructor, optimizer; repair_periods=10, kwargs...)
-    initial_restoration = replicate_restoration_network(network_data, count=repair_periods)
-    final_restoration =  Dict{String,Any}("nw" => Dict{String,Any}())
-    final_restoration["multinetwork"] = true
-
-    for k in keys(initial_restoration)
-        if k != "nw"
-            final_restoration[k] = initial_restoration[k]
-        end
+function run_iterative_restoration(network, model_constructor, optimizer; repair_periods=2, kwargs...)
```

Comment:
If you like we can set up a VC to talk about this algorithm in detail.  I can't really comment on its correctness or simplifications without talking through key goals and assumptions.

We should also need to think carefully about when new features move into the `master` release as the maintenance burden is increasing now that this is a registered package that is being integrated into our production code.